Jellyfin.Api/Controllers/AttachmentsController.cs

```diff
@@ -0,0 +1,85 @@
+#nullable enable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Extensions;
+using MediaBrowser.Controller.Library;
+using MediaBrowser.Controller.MediaEncoding;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Jellyfin.Api.Controllers
+{
+    /// <summary>
+    /// Attachments controller.
+    /// </summary>
+    [Route("Videos")]
```

Comment:
Uri is usually `[controller]` so this is a bit weird to me. We should change it at some point. A single VideosController

Reply:
I agree, but I believe the idea is to get  a 1 to 1 copy first